crates/rolldown/tests/rolldown/topics/deconflict/wrapped_esm_default_function/artifacts.snap

```diff
@@ -42,7 +41,7 @@ init_foo();
 
 ```
 - ../foo.js
-(4:15) "function " --> (19:0) "function "
+(4:0) "export default function " --> (19:0) "function "
```

Comment:
![image](https://github.com/user-attachments/assets/288c9fa7-08b2-43f8-89c9-007f2c57cd20)
it is expected, though.